src/c_sharp_graph/query.rs

```diff
@@ -3,32 +3,36 @@ use std::{
     vec,
 };
 
-use crate::c_sharp_graph::results::{Location, Position, Result};
+use crate::c_sharp_graph::{
+    loader::SourceType,
+    results::{Location, Position, ResultNode},
```

Comment:
Just wanted to say thank you for using `ResultNode`, my IDE was painfully trying to resolve to the standard `Result` type every time :joy: